python/interpret_community/explanation/explanation.py

```diff
@@ -248,11 +251,20 @@ def features(self):
     def is_raw(self):
```

Comment:
one thing to think about is we may want to eventually make this a list of attributes instead of several Boolean properties on the class, but we won't be able to make this breaking change at this point in time

Reply:
Will plan to incorporate whenever we can make breaking changes!